Khan/perseus
# Use Wonder Blocks in InteractiveGraphEditor

## Summary:

In preparation for adding a wonder blocks dropdown for adding a locked figure, 
I'm making some other components in InteractiveGraphEditor use wonder blocks
as well for the sake of consistency an accessibility.

- Use wonder-blocks-dropdown SingleSelect instead of `<select>`
- Update PropCheckBox so it uses WB Checkbox
- Use WB PhosphorIcon instead of a custom image that was indiscernible anyway 
- Use WB Typography's LabelSmall everywhere
- Update labels so they work programmatically (clicking on them also highlights/clicks on the element that the label applies to)
- Move `correct answer` down by the graph while I'm here. Requested by content authors.

Note: I don't think WB dropdown has any equivalent for `<optgroup>` yet. In the
meantime, I'm just using a horizontal rule to separate the metric and imperial
units in the "Ruler label" dropdown.

Issue: https://khanacademy.atlassian.net/browse/LC-1702

## Test plan:
`yarn jest`

Storybook
- Go to the EditorPage story http://localhost:6006/?path=/docs/perseus-editor-editorpage--docs
- Click the "Add a widget" dropdown
- Select "Interactive graph"
- Open the new "interactive-graph 1" dropdown
- Check that the checkboxes are wonder blocks checkboxes
- Confirm that the "Type of graph" dropdown is a wonder blocks dropdown
- Click the "Show ruler" checkbox
- Confirm that the "Ruler label" and "Ruler ticks" dropdowns are wonder blocks dropdowns

### Screenshot

Before is on the left. After is on the right.

<img width="862" alt="Screenshot 2024-03-04 at 6 05 36 PM" src="https://github.com/Khan/perseus/assets/13231763/b1f71709-7f5d-48c9-994d-3fc4e7869279">
